packages/web-app/src/modules/storefront-views/blocks/StorefrontHeroBlock.tsx

```diff
@@ -0,0 +1,38 @@
+import { rewardRoute } from '../../../RouteUtils'
+import { StorefrontHeroBlockProps } from '../../storefront/models'
+import { StorefrontHero } from '../components/StorefrontHero'
+import { StorefrontHeroItem } from '../components/StorefrontHeroItem'
+
+interface Props {
+  block: StorefrontHeroBlockProps
+}
+
+export const StorefrontHeroBlock = ({ block }: Props) => {
+  return (
+    <StorefrontHero key={block.id} title={block.title}>
+      {block.items.map((hero) => {
+        const heading = hero.reward?.name || hero.heading
+        const subHeading = hero.reward?.price ? `$${hero.reward?.price.toFixed(2)}` : hero.subheading
```

Comment:
These are flipped... also, converting `hero.reward?.price` to a truthy/falsy value doesn't work when the price is free. For consistency, shouldn't a free price be transformed into a "FREE" string like in the reward list?

```suggestion
        const subHeading = hero.subheading || `$${hero.reward?.price.toFixed(2)}`
```